weaviate/graphql/nearvectorbuilder_test.go

```diff
@@ -79,9 +79,32 @@ func TestNearMultiVectorBuilder_build(t *testing.T) {
 		require.Contains(t, str, "vectorPerTarget: ")
 		require.Contains(t, str, "one: [1,2,3]")
 		require.Contains(t, str, "two: [4,5,6]")
-		require.Contains(t, str, "targetVectors:")
-		require.Contains(t, str, "\"one\"")
+		require.NotContains(t, str, "vector:")
+		require.NotContains(t, str, "combinationMethod:")
+		require.NotContains(t, str, "weights:")
+	})
+
+	t.Run("No combination with multiple vectors per target", func(t *testing.T) {
+		vector := NearVectorArgumentBuilder{}
+		str := vector.WithVectorsPerTarget(map[string][][]float32{"one": {{1, 2, 3}, {7, 8, 9}}, "two": {{4, 5, 6}}}).build()
+		require.Contains(t, str, "vectorPerTarget: ")
+		require.Contains(t, str, "one: [[1,2,3],[7,8,9]]")
+		require.Contains(t, str, "two: [4,5,6]")
```

Comment:
Will the server be able to handle `[[4,5,6]]` correctly? Like see that it's just 1 vector and not do any extra work.
Always sending `[][]float32` could simplify the code by removing [this special case](https://github.com/weaviate/weaviate-go-client/pull/239/files#diff-f350dc094391aa08961984075cbc93460640cdcbdaeeacdb538dfd5dbc7e4154R87-R98).

Reply:
yep, works 👍  Removed the extra handling